libraries/AP_NavEKF/EKF_Buffer.cpp

```diff
@@ -10,22 +10,21 @@
 
 // constructor
 ekf_ring_buffer::ekf_ring_buffer(uint8_t _elsize) :
-    elsize(_elsize)
+    elsize(_elsize),
+    buffer(nullptr)
```

Comment:
> Why the initialisation here?

because I declare on the stack in the test suite, and {} doesn't work as the constructor overrides, so it dies when it tries to free a non-null ptr on reset